.github/workflows/test.yml

```diff
@@ -12,23 +12,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Cache Memgraph Docker image
-      id: cache-memgraph-docker
-      uses: actions/cache@v1
-      with:
-        path: ~/memgraph
-        key: cache-memgraph-v${{ env.MG_VERSION }}-docker-image
-    - name: Download Memgraph Docker image
-      if: steps.cache-memgraph-docker.outputs.cache-hit != 'true'
-      run: |
-        mkdir ~/memgraph
-        curl -L https://memgraph.com/download/memgraph/v${{ env.MG_VERSION }}/docker/memgraph-${{ env.MG_VERSION }}-community-docker.tar.gz > ~/memgraph/memgraph-docker.tar.gz
-    - name: Load Memgraph Docker image
-      run: |
-        docker load -i ~/memgraph/memgraph-docker.tar.gz
     - name: Build project
       run: |
-        docker build -t memgraph:mage .
```

Comment:
@jmatak was this a bug? Why are we here building `memgraph:mage` and we are deploying `memgraph-mage`?